extensions/positron-assistant/src/models.ts

```diff
@@ -97,6 +105,25 @@ abstract class AILanguageModel implements positron.ai.LanguageModelChatProvider
 		this.provider = _config.provider;
 	}
 
+	async resolveConnection(token: vscode.CancellationToken): Promise<boolean> {
+		token.onCancellationRequested(() => {
+			return false;
+		});
+
+		try {
+			// send a test message to the model
+			await ai.generateText({
+				model: this.model,
+				prompt: 'Hello!',
```

Comment:
This can return a pretty wordy response with some models and since each token takes some time to generate, I think we should use a prompt that is likely to be faster. Maybe like this?

<img width="234" alt="image" src="https://github.com/user-attachments/assets/2c84773c-f9b0-463f-a34d-83910776c998" />